content/en/tracing/setup/nodejs.md

```diff
@@ -94,7 +101,9 @@ For details about how to how to toggle and configure plugins, check out the [API
 
 | Module           | Versions | Support Type    | Notes                                      |
 |------------------|----------|-----------------|--------------------------------------------|
+| [connect][45]    | `>=2`    | Fully Supported |                                            |
```

Comment:
```suggestion
| [connect][45]    | `>=2`    | Fully Supported | Requires `dd-trace-js` v0.13.0+                      |
```

Reply:
As discussed, let's leave this out for now since it would be difficult to track the exact library version support by tracer version.